blueprint/src/chapters/stochastic_integral.tex

```diff
@@ -9,6 +10,113 @@ \section{Total variation and Lebesgue-Stieltjes integral}
 We need to investigate what's already in Mathlib. See \texttt{Mathlib.Topology.EMetricSpace.BoundedVariation}.
 
 
+\section{Doob's Lp inequality}
+
+In this section, we prove Doob's Lp inequality.
+
+\begin{lemma}\label{lem:convex_of_mg_is_submg}
+  Let $X:T\times\Omega\rightarrow \mathbb{R}^d$ a martingale.
+  Let $\phi:\mathbb{R^d}\rightarrow \mathbb{R}$ convex such that
+  $\phi(X_t)\in L^1(\Omega)$ for every $t\in T$. Then $\phi(X)$ is a sub-martingale.
+\end{lemma}
+\begin{proof}
+  % See 1.4.12 Pascucci
+  By Jensen
+  $\phi(X_t) = \phi\left( \mathbb{E}[X_T\ |\ \mathcal{F}_t] \right)\leq \mathbb{E}[\phi(X_T)\ |\ \mathcal{F}_t]$.
```

Comment:
This is conditional Jensen, which is not in Mathlib yet, but is in a PR (#27953, which I need to review). Can you add that ref?